version_control/katalyst_by_codurance/css/templates/home.css

```
@@ -210,4 +212,26 @@ img.methodology__graphic {
     margin-top: var(--space-l);
     max-width: 48rem;
     margin-inline: auto;
-}
\ No newline at end of file
+}
+
+/* Highlight element */
+
+.highlight {
+    position: relative;
```

I've seen one small issue regarding the positioning of the highlight span: when used on sections with background the :after element gets beneath it and isn't visible. It seems to be fixed just by adding a z-index:0 here, but if you think of anything better just go ahead!
```suggestion
    position: relative;
    z-index: 0;
```